src/components/calendar/calendar.component.scss

```diff
@@ -1,33 +1,43 @@
 @import "colors/index";
 
-$calendar-bg: $color-blue-grey-600;
-$calendar-title-bg: $color-blue-grey-700;
+$calendar-bg: $color-blue-grey-800;
+$calender-border: #2D3544;
```

Comment:
Can we replace these with existing colors? If the exact ones are not present, let's use the closest ones.